tests/test_backward_compatibility.py

```diff
@@ -62,21 +63,22 @@ def setUpClass(cls) -> None:
             (
                 "async_mode_enabled",  # test name
                 True,  # async consumer flag
-                AsyncConsumer,  # expected consumer instance
+                SelectConsumerThread,  # expected consumer instance
             ),
             (
                 "async_mode_disabled",
                 False,
-                ConsumerThread,
+                BlockingConsumerThread,
             )
         ]
     )
     def test_stable_register_consumer_args(
         self,
-        test_name: str,
-        async_mode_enabled: bool,
-        expected_consumer_instance
+        test_name,
```

Comment:
From [parameterized docs on PyPI](https://pypi.org/project/parameterized/), should this be `name`?

Is there a simple method for confirming that both test cases are being run in GHA? I only see that each test case passed, not anything to indicate the different parameterized inputs were tested.

Reply:
> From [parameterized docs on PyPI](https://pypi.org/project/parameterized/), should this be `name`?
> 
> Is there a simple method for confirming that both test cases are being run in GHA? I only see that each test case passed, not anything to indicate the different parameterized inputs were tested.

Yeah, we may use either `name` or `test_name`, both are resolved correctly by the parameterized. You may download and lookup backward compatibility test artifacts for any python version there is a note that both of those tests ran and succeeded.

This is sample artifact for Python 3.10:

`<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="0" skipped="0" tests="2" time="0.760" timestamp="2024-11-28T19:51:41.294575" hostname="fv-az1427-607">
        <testcase classname="tests.test_backward_compatibility.TestBackwardCompatibility" name="test_stable_register_consumer_args_0_async_mode_enabled" time="0.060" />
         <testcase classname="tests.test_backward_compatibility.TestBackwardCompatibility" name="test_stable_register_consumer_args_1_async_mode_disabled" time="0.442" /></testsuite></testsuites>`